example-app/src/main/java/com/adyen/checkout/example/ui/card/SessionsCardTakenOverViewModel.kt

```diff
@@ -114,6 +116,8 @@ internal class SessionsCardTakenOverViewModel @Inject constructor(
                     ?: throw IllegalStateException("Return url should be set"),
                 shopperEmail = keyValueStorage.getShopperEmail(),
                 allowedPaymentMethods = listOf(paymentMethodType),
+                storePaymentMethodMode = StorePaymentMethodMode.ASK_FOR_CONSENT.mode,
```

Comment:
Instead of manually passing these values every time we need them, how about we add them as default values to the parameters of the `getSessionRequest` method?